src/Lean/PrettyPrinter/Delaborator/Basic.lean

```diff
@@ -36,19 +36,25 @@ The delaborator is extensible via the `[delab]` attribute.
 namespace Lean.PrettyPrinter.Delaborator
 
 open Lean.Meta SubExpr
+open Lean.Elab (Info TermInfo Info.ofTermInfo)
 
 structure Context where
   defaultOptions : Options
   optionsPerPos  : OptionsPerPos
   currNamespace  : Name
   openDecls      : List OpenDecl
   inPattern      : Bool := false -- true whe delaborating `match` patterns
+  subExpr        : SubExpr
+
+structure State where
```

Comment:
Could you please comment this ner structure?

Reply:
I wasn't sure what to say about `State` but I commented the fields.